bin/server.js

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env node
+var debug             = require('debug')('github:server');
+var base              = require('taskcluster-base');
+var api               = require('../lib/api');
+var path              = require('path');
+var Promise           = require('promise');
+var exchanges         = require('../lib/exchanges');
+var _                 = require('lodash');
+
+/** Launch server */
+var launch = async function(profile, publisher) {
```

Comment:
I have end to end tests here in ``test/taskcluster_github_api_integration_test.js`` it's just that they only activate when pulse credentials are available. I agree that actually testing pulse/github in integration tests is important, but imo there's value in being able to checkout a project and run its unit tests without needing any outside credentials.